actpool/actpool.go

```diff
@@ -180,7 +180,7 @@ func (ap *actPool) Start(ctx context.Context) error {
 	sort.Sort(blobs)
 	for _, selp := range blobs {
 		if err := ap.add(ctx, selp); err != nil {
-			return err
+			log.L().Info("Failed to load action from store", zap.Error(err))
```

Comment:
what's the error happening now? should we fix that, for actionstore to function correctly?

Reply:
the actions storing in actstore is not that important, it is fine to ignore them